### DISCUSSION TITLE
Permission Denied with SSH Despite Correct Keys and Passphrase

### DISCUSSION THREAD

**Discussion by tygre-chingu** [Hypfer/Valetudo#1792] [Q&A / Support]
Hi!

Long time ago, I rooted and installed Valetudo on my Xiaomi Mi vaccum robot. It worked great until recently I noticed that the robot IP address changed (never a good sign!), which went from 192.168.0.210 to 192.168.0.196. I wanted to SSH into the robot to investigate but I receive the error `Permission denied, please try again.`. I **can** connect via HTTP though. I looked "everywhere", [including this recent discussion](https://github.com/Hypfer/Valetudo/discussions/1773) but this solution doesn't work for me.

What I've checked: I already tried `ssh -vv -o PubkeyAcceptedKeyTypes=+ssh-rsa -i ~/xiaomivacuumkey 192.168.0.196` as suggested in the other question. I have the correct passphrase and public/private keys pairs. I regenerated different keys with different algorithms: RSA and ED25519. What I see when `ssh -vv -i xiaomivacuumkey 192.168.0.196`, no matter the algorithms used to generate the key, is the "error" message `send_pubkey_test: no mutual signature algorithm`. 

What am I missing?
(Sorry if this is a duplicate, I posted my question before but then it disappeared from the list.)

**Comment by wailnucleus** [comment]
Please provide the output of your connection attempt:

ssh -vvv -i ~/xiaomivacuumkey 192.168.0.196

**Comment by tygre-chingu** [comment]
Thank you for your help, @wailnucleus!
I attach the [output](https://github.com/Hypfer/Valetudo/files/11359789/Output.txt), until SSH ask me to enter a password. (No matter the password, the connection always fails.)
